jekyll/_cci2/language-javascript.md

```diff
@@ -174,21 +169,20 @@ workflows:
 ### 5. Conclusion
 {: #conclusion }
 
-Success! You just set up a Node.js app to build on CircleCI. Check out your project’s [pipeline page]({{site.baseurl}}/2.0/project-build/#overview) to see how this looks when building on CircleCI.
+You just set up a Node.js app to build on CircleCI. Check out your project’s [pipeline page]({{site.baseurl}}/2.0/project-build/#overview) to see how this looks when building on CircleCI.
 
 ## Full configuration file
```

Comment:
Currently the javascript project we link to has a `master` branch rather than `main` but I have requested this be updated for consistency.